### PR TITLE
Adding asyncio docker builder

### PR DESCRIPTION
**Pull request by JulianGindi**

### CODE REVIEW

File: build-api/api.py [hunk 1]
```diff
@@ -1,11 +1,53 @@
...
+import os, shutil, subprocess, asyncio
 
 from flask import Flask, jsonify, render_template, request
 
 app = Flask(__name__)
 
+def build_docker_container():
+    # Spin up the docker container to pull and build the project
+    startscript = subprocess.Popen('docker build --no-cache=True --tag="franklin_builder:tmp" .',
+                                   cwd='tmp',
```
Comment (oudeismetis) [inline build-api/api.py]: andrew says pep8

File: build-api/api.py [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
+    shutil.rmtree(tmp_dir)
+
+
+def schedule_coroutine(target, *, loop=None):
+    """Schedules target coroutine in the given event loop
+    If not given, *loop* defaults to the current thread's event loop
+    Returns the scheduled task.
+    """
+    if asyncio.iscoroutine(target):
+        return asyncio.ensure_future(target, loop=loop)
```
Comment (oudeismetis) [inline build-api/api.py]: not used?

**Comment by oudeismetis** [comment]
:+1:

**Comment by oudeismetis** [comment]
:sparkles: for pep8

**Comment by CorneliusIV** [comment]
:+1:

**Pull request merged**